zuul/connection/github.py

```diff
@@ -185,6 +185,20 @@ def _event_pull_request_review(self, request):
         event.type = 'pr-review'
         return event
 
+    def _event_status(self, request):
+        body = request.json_body
+        action = body.get('action')
+        if action == 'pending':
+            return
```

Comment:
We could, tho I had our use case described in the flowchart at [1] in mind while doing this.  We could certainly let 'pending' events through but that'd result in some expensive API calls later on in order to make the scheduling decision.  

[1] https://github.com/BonnyCI/lore/blob/master/developers/contributing/designs/github-pr-check-gate.md